### ISSUE TITLE
quizzes not working

### ISSUE THREAD

**Issue by AinzOoalGown-fun** [Tom60chat/Microsoft-Rewards-Farmer-Sharp#45]
quizzes are not work. only  automate bing searches pc and mobile work.

<details><summary>Log file</summary>
[Log] 
§cMicrosoft §aRewards §bFarmer §eby §rTom Olivier                                   Version 0.9.1.0 (Checking for new versions...)

[Log] 
§cMicrosoft §aRewards §bFarmer §eby §rTom Olivier                                                 Version 0.9.1.0 (Latest version)

[Log] 
[0](0/6) - 0 points: Restauration de la session...

[Log] 
[0](1/6) - 0 points: Obtention du nombre de points gagnés...

[Log] 
[0](2/6) - 198 points: Obtention des cartes...

[Log] 
[0](2/6) - 198 points: Traitement des cartes 0 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 1 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 2 / 3

[Log] 
[0](2/6) - 198 points: Obtention des cartes...

[Log] 
[0](2/6) - 198 points: Traitement des cartes 0 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 1 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 2 / 3

[Log] 
[0](2/6) - 198 points: Obtention des cartes...

[Log] 
[0](2/6) - 198 points: Traitement des cartes 0 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 1 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 2 / 3

[Log] 
[0](2/6) - 198 points: Obtention des cartes...

[Log] 
[0](2/6) - 198 points: Traitement des cartes 0 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 1 / 3

[Log] 
[0](2/6) - 198 points: Traitement des cartes 2 / 3

[Log] 
[0](5/6) - 198 points: Obtention du nombre de points gagnés...

[Log] 
[0](6/6) - 198 points: Terminer - Gain : 0 - Total : 198
</details>

Please fix the problem.
Thank you -_-

**Comment by Tom60chat** [comment]
Hello, please use version 0.9.2

**Comment by Tom60chat** [comment]
Yeah, maybe wait for the new version and please don't copy and paste the log file, but drag and drop it inside your comment.

**Comment by Tom60chat** [comment]
The new version 0.9.3 has been released.

Please test it and leave a comment if your problem has been solved or not.

[Download link](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/releases/tag/v0.9.3-beta1)

**Comment by AinzOoalGown-fun** [comment]
it's very long, and it's worse.
[2023-01-07 20.09.13.txt](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/files/10367045/2023-01-07.20.09.13.txt)

![Capture d’écran 2023-01-07 202348](https://user-images.githubusercontent.com/73220716/211167300-f15d0cab-63f9-4923-9085-f8daacf63d09.png)

Thank You -_-

**Comment by Tom60chat** [comment]
Are you using a VPN ?

**Comment by AinzOoalGown-fun** [comment]
no

**Comment by Tom60chat** [comment]
Okay, I was able to reproduce the issue on my side.
This will be fixed on the new version.

**Comment by laoka3** [comment]
Hi there, don't know why, since this week, the daily quiz is not working any more. The rests are working

**Comment by Tom60chat** [comment]
Normal, this is a know issue and is still being fixed or has been fixed but not released yet.

**Comment by AinzOoalGown-fun** [comment]
> my

Thank you -_-  I will wait.

**Comment by AinzOoalGown-fun** [comment]
> Okay, I was able to reproduce the issue on my side. This will be fixed on the new version.

Thank you -_- I will wait.

**Comment by demiryuca** [comment]
When will the new version arrive? I'm excited to see the bug fixes because there are a lot.

**Comment by demiryuca** [comment]
In the V0.10.0 program still struggles to complete the quizzes below, its like repeating the getting cards and processing again and again.

<img width="545" alt="image" src="https://user-images.githubusercontent.com/52105261/221405900-81ebeb79-b6e8-4989-9df0-86fee2df73f9.png">


> Okay, I was able to reproduce the issue on my side. This will be fixed on the new version.

**Comment by Tom60chat** [comment]
Hello, I hope I haven't kept you waiting too long.

Version 0.10 is out, I invite you to test this version.

You can get it at the following address: https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/releases/tag/v0.10.0-beta1

If your problem has been solved, I encourage you to close this issue, you can reopen it if you wish.

**Comment by AinzOoalGown-fun** [comment]
> Hello, I hope I haven't kept you waiting too long.
> 
> Version 0.10 is out, I invite you to test this version.
> 
> You can get it at the following address: https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/releases/tag/v0.10.0-beta1
> 
> If your problem has been solved, I encourage you to close this issue, you can reopen it if you wish.

Hello -_-  Thanks you for releasing a new version.
a new problem with the last version 0.10

[2023-03-16 03.36.41.txt](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/files/10986114/2023-03-16.03.36.41.txt)

Thank you

**Comment by AinzOoalGown-fun** [comment]
> 



> Hello, I hope I haven't kept you waiting too long.
> 
> Version 0.10 is out, I invite you to test this version.
> 
> You can get it at the following address: https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/releases/tag/v0.10.0-beta1
> 
> If your problem has been solved, I encourage you to close this issue, you can reopen it if you wish.

Hello , Thank you for the new version
Works well, the program doesn't do everything, you must restart it a second time to do everything. 
It solves the problems of daily quizzes

There are extra quizzes that come at holiday time like on the pictures that the program doesn't do.

![quizz](https://user-images.githubusercontent.com/73220716/225509407-004fbedd-2df9-412d-9af4-f57ce40b4eb7.png)
![quizz2](https://user-images.githubusercontent.com/73220716/225509409-89e64132-6c33-4ec8-bde0-5b11502445d2.png)

This version is very good thank you, except a bug crash where you have to restart the program to finish everything.

[crash.txt](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/files/10986433/crash.txt)

**Comment by Tom60chat** [comment]
Some card, has not been implemented properly yet.

For the extra quizzes, this idea it's on discord, will not be implemented yet, because i'm focusing on stability for now.

For the crash go to #67

**Comment by AinzOoalGown-fun** [comment]
I understood, thank you -_-

**Comment by Tom60chat** [comment]
> Works well, the program doesn't do everything, you must restart it a second time to do everything.
Just reopening this until most of unprocessed cards that is currently implement are actually fixed 😁

**Comment by K1ngJony** [comment]
It seems the bug has returned in v0.10.1.0...

![image](https://user-images.githubusercontent.com/45821611/233217317-1ef41c21-25e7-4f4c-99e6-92b4c19ded49.png)

[2023-04-20 08.45.42.txt](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/files/11278115/2023-04-20.08.45.42.txt)

Log is of the second run, attempting to complete the cards.
